package.json

```diff
@@ -2,31 +2,16 @@
   "name": "yargs",
   "version": "17.7.2",
   "description": "yargs the modern, pirate-themed, successor to optimist.",
-  "main": "./index.cjs",
+  "main": "./index.mjs",
   "exports": {
     "./package.json": "./package.json",
-    ".": [
-      {
-        "import": "./index.mjs",
-        "require": "./index.cjs"
-      },
-      "./index.cjs"
-    ],
-    "./helpers": {
-      "import": "./helpers/helpers.mjs",
-      "require": "./helpers/index.js"
-    },
+    "./helpers": "./helpers/helpers.mjs",
     "./browser": {
       "import": "./browser.mjs",
       "types": "./browser.d.ts"
     },
-    "./yargs": [
-      {
-        "import": "./yargs.mjs",
-        "require": "./yargs"
-      },
-      "./yargs"
-    ]
+    ".": "./index.mjs",
+    "./yargs": "./index.mjs"
```

Comment:
What is the short or long term intent with require/import `yargs/yargs`? Is this just for backwards compatibility and could be removed from examples, or stay around for a while?

Reply:
My thought was it's just cheap to add this export for backwards compatibility. Stops us from breaking people who followed old documentation.

Reply:
Definitely want it for backwards compatibility, and I liked it did not need a file. Nice "trick".

I was mainly wondering if `yargs/yargs` will be disappearing from customer facing docs.

Reply:
I'll remove any references I see to `yargs/yargs` in docs as part of this PR 👍